Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -1821,6 +1833,8 @@ var surfaceShaderSetOptionsScratch = {
   colorToAlpha: undefined,
 };
 
+var defaultUndergroundColorByDistance = new NearFarScalar();
```

Comment:
consider adding a defaultUndergroundColor as well (for clarity), either here or down below